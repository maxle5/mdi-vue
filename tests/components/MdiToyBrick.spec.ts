
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiToyBrick from "../../src/components/MdiToyBrick.vue";

test("MdiToyBrick snapshot", () => {
  const wrapper = mount(MdiToyBrick, {});
  expect(wrapper.html()).toMatchSnapshot();
});
