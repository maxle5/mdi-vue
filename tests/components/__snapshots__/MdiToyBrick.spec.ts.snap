// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiToyBrick snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M19 6V5A2 2 0 0 0 17 3H15A2 2 0 0 0 13 5V6H11V5A2 2 0 0 0 9 3H7A2 2 0 0 0 5 5V6H3V20H21V6Z" style="fill: currentColor;"></path>
</svg>"
`;
